raiden-dapp/src/views/DisclaimerRoute.vue

```diff
@@ -1,10 +1,52 @@
 <template>
   <div data-cy="disclaimer" class="disclaimer">
     <div class="disclaimer__content">
-      <div class="disclaimer__content__paragraphs font-weight-light">
-        <p v-for="(paragraph, index) in $t('disclaimer.paragraphs')" :key="index">
-          {{ paragraph }}
-        </p>
+      <div class="disclaimer__content__text">
+        <span class="disclaimer__content__text__header">
+          {{ $t('disclaimer.user-info.header') }}
+        </span>
+        <span class="disclaimer__content__text__body">{{ $t('disclaimer.user-info.body') }}</span>
+        <div>
+          <ul
+            v-for="(bulletPoint, index) in $t('disclaimer.user-info.bullet-points')"
+            :key="index"
+            class="disclaimer__content__text__bullet-points"
+          >
+            <li>
+              {{ bulletPoint.text }}
+              <a :href="disclaimerUrlMappings[bulletPoint.url]" target="_blank">
+                {{ bulletPoint.link }}
```

Comment:
Suggestion (and I think we actually discussed this solution together):
I think the `url` property is quite confusing. From within the translations this just makes no sense. You read it and don't know it is a special key that gets converted. As you have written it, the `url` is actually a proper translation which could differ for all languages. I would like to (re)suggest to convert the array into an object and use just the names.
So in the locals this would looks like:

```js
bullet-points: {
   portal: {
     text: "foo",
     link: "bar",
   },
   docs: {
     ...
   },
   ... 
}
```

And then in the template you can just do `v-for="(bulletPoint, name) in $t(...`.
In fact this only changes a little to your current implementation and mapping in the component. But I personally think that these subtile changes represent a better structure. It has a stronger separation of translation and data. Although the linking works most-likely the same.

---

If you still prefer your approach and want to stick with it, please at least rename the `url` translation key as it is quite confusing. But maybe I'm the only one...

Reply:
I can make it an object instead, no problem. :)